### PR TITLE
fra + frp

### PR DESCRIPTION
**Pull request by hectoralos**
Added the missing strings in fra.json and the new frp language (Arpitan/Francoprovençal) (new try)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,46 @@
...
-a live copy of Html-tools using `config.conf.example`.
-
-[1]: http://wiki.apertium.org/wiki/Apertium-html-tools
-[2]: http://apertium.org
-[3]: http://wiki.apertium.org/wiki/Apertium-apy
-[4]: http://wiki.apertium.org/wiki/ScaleMT
-[5]: http://wiki.apertium.org/wiki/Apertium-html-tools
-[6]: https://www.docker.com/
-[7]: https://circleci.com/
+Localization
```
Comment (sushain97) [inline README.md:1]: This change is to the incorrect README?

Comment (hectoralos) [reply README.md:1]: Really, I got lost. I don't understand anything. I can't imagine what is happening. I'm uploading the files where I get something in "git diff", and in this particular file I see changes only in 3 lines. I better close everything, and send to you the two files with the strings by email. We are both wasting a lot of time in something that seems to be (and should be) extremely easy and mechanical.

Comment (sushain97) [reply README.md:1]: >  I'm uploading the files

This is the crux of the issue. The recommended workflow is to push your branch to GitHub via `git push` and then GitHub will provide a link to create a pull request.

Comment (hectoralos) [reply README.md:1]: It does not seem GitHub provides any link to me :

```
$ git add frp.json 

$ git commit README.md fra.json frp.json -m "+frp & actualisation fra"
[master 2054055] +frp & actualisation fra
 3 files changed, 135 insertions(+), 68 deletions(-)
 rewrite assets/strings/fra.json (61%)
 create mode 100644 assets/strings/frp.json

$ git push
Username for 'https://github.com': hectoralos
Password for 'https://hectoralos@github.com': 
Comptant els objectes: 7, fet.
Delta compression using up to 8 threads.
Comprimint objectes: 100% (6/6), fet.
Escrivint els objectes: 100% (7/7), 3.86 KiB | 0 bytes/s, fet.
Total 7 (delta 5), reused 2 (delta 1)
remote: Resolving deltas: 100% (5/5), completed with 5 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/master.
remote: error: You're not authorized to push to this branch. Visit https://docs.github.com/articles/about-protected-branches/ for more information.
To https://github.com/apertium/apertium-html-tools/
 ! [remote rejected] master -> master (protected branch hook declined)
error: s'ha fallat en pujar algunes referències a 'https://github.com/apertium/apertium-html-tools/'
```

**Pull request closed without merging**